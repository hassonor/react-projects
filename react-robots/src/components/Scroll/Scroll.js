const Scroll = (props) => {
  return (
    <div
      style={{
        overflowY: "scroll",
        border: "4px solid black",
        height: "800px",
      }}
    >
      {props.children}
    </div>
  );
};

export default Scroll;
