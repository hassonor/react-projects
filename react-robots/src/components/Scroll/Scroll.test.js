import React from "react";
import { shallow } from "enzyme";
import Scroll from "./Scroll";

it("expect to render Scroll Component", () => {
  expect(shallow(<Scroll />)).toMatchSnapshot();
});
