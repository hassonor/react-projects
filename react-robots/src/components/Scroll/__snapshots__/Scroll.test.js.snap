// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Scroll Component 1`] = `
<div
  style={
    Object {
      "border": "4px solid black",
      "height": "800px",
      "overflowY": "scroll",
    }
  }
/>
`;
